pkg/cmd/attestation/verify/verify.go

```diff
@@ -54,27 +58,38 @@ func NewVerifyCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command
 			To see the full results that are generated upon successful verification, i.e.
 			for use with a policy engine, provide the %[1]s--format=json%[1]s flag.
 
-			The attestation's certificate's Subject Alternative Name (SAN) identifies the entity
-			responsible for creating the attestation, which most of the time will be a GitHub
-			Actions workflow file located inside your repository. By default, this command uses
+			The signer workflow's identity is validated against the attestation's
+			certificate's Subject Alternative Name (SAN). Often, the signer workflow is the
```

Comment:
I think this might be a simpler way of saying this:

```suggestion
			The signer workflow's identity is validated against the Subject Alternative Name (SAN)
			within the attestation certificate. Often, the signer workflow is the
```